src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -238,6 +238,8 @@
     private static final String PREFIX_THREEDS = "threeds2.";
     private static final String PREFIX_THREEDS_RESPONSE_DATA = "threeds2.threeDS2ResponseData.";
 
+    private static final List<String> ADDITIONAL_PROPERTIES_TO_PERSIST = ImmutableList.of("3ds2.csrf", "3ds2.resUrl", "3ds2.formUrl");
```

Comment:
These are groupon specific

Reply:
Addressed in 60db2b5 to specify these params in the config file.